common/config.go

```diff
@@ -0,0 +1,69 @@
+package common
+
+import (
+	"fmt"
+
+	"github.com/go-ini/ini"
+)
+
+//
+type DefaultConfigProvider struct {
+	data string
+}
+
+func NewDefaultConfigProvider() *DefaultConfigProvider {
+	d := new(DefaultConfigProvider)
+	d.data = `
+[Debug]
+# Randomize adds a random factor +/- the give percent.  3.1 for 3.1%
+  Randomize=0.1
```

Comment:
why is this the default?

Reply:
Unit tests, it won't be created and will still panic if you delete the .ini file

Reply:
@WhoSoup I needed a way to grab the default.ini from the unit tests without needing to deal with file pathing. I grabbed this from the default ini in the repo.

Reply:
> Unit tests, it won't be created and will still panic if you delete the .ini file

What do you mean by this? Will my unit tests panic under some config conditions?

Reply:
I renamed it to be more clear